### ISSUE TITLE
Will the code of  pytorch version be released?

### ISSUE THREAD

**Issue by zhao1iang** [malllabiisc/HyTE#2]
Thanks for your outstanding work, and I want to know if you have the plan to release pytorch version code?

**Comment by ssdasgupta** [comment]
Hi @zhao1iang , thanks for showing interest in this work. The code only requires tensorflow to run gradient decent optimiser and to lookup the embeddings of entity and relations from the embedding matrix. In my opinion the platform will not be a barrier since no complex architecture is involved. In case you need any help to code it in pytorch then I can help for sure. You can also drop me a personal mail regarding this.
